### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by iamwillow** [DrewSSP/delete_phantom_answers#2]
It's been quite a while since the last time I ran this code. 
When running the command in the terminal, this fancy error is appearing:

```
Traceback (most recent call last):
  File "main.py", line 49, in <module>
    number_of_pages = int(html.fromstring(requests.get(course_database_url, cookies=cookies).content).xpath("//div[contains(@class, 'pagination')]/ul/li")[1].text_content())
IndexError: list index out of range
```

Any ideas on where to start bug testing?
- Python 3.6.1
- Requests 2.11.1
- Sys ??
- Multiprocessing 2.6.2.1
- Lxml 3.6.4

**Comment by DrewSSP** [comment]
The most likely cause of the issue is that your cookies expired over the time that you haven't been using the script and Memrise assigned new ones to you. This cause the script to fail the authentication process forcing Memrise to respond with the "you don't have access to this course's database" page. 

The reason that this appears as an index error is because the entire database page is processed in the script as a giant list. That list is supposed to include the number of pages that are in your course database, which of course doesn't exist if Memrise isn't giving permission for the script to see your course's database page. IndexError is just Python's way of saying "you tried to find something in the list and it wasn't there".

Long story short, delete your variables.py file and make it again using the instructions provided with the script.

**Comment by iamwillow** [comment]
Thanks for the detailed reply. In that case, I'm going to presume that I've incorrectly set up my variables.py file as this is a new course with fresh cookies.

For locating this bug, let me fill you in on how I've created the file. 

**Cookies taken from:**
https://www.memrise.com/course/1484373/advanced-english-for-native-speakers/edit/database/2468388/
- This is the course database page

**Variable not found:**
_sp_id.7bc7
- I left this as the default from the README file as it was not listed as one of the cookies.

Do either of these steps appear to be incorrect?

**Comment by DrewSSP** [comment]
You're on the right track but there is no need to modify the variables file to suit the script's example cookies. If Memrise doesn't use a _sp_id.7bc7 then don't include it. If there are any extra variables that Memrise uses in its cookies these days that are not found in my example cookies then include those. Memrise decides what the cookies look like, not the script.
